ui/selectors/selectors.js

```diff
@@ -49,6 +49,7 @@ import {
   HardwareTransportStates,
 } from '../../shared/constants/hardware-wallets';
 import { KeyringType } from '../../shared/constants/keyring';
+import sharedSelectors from '../../shared/modules/selectors';
```

Comment:
As mentioned in the selector file, we can just export them individually and then allow single imports here for only the functions needed?